apps/condo/domains/analytics/hooks/useDashboardFilters.tsx

```diff
@@ -82,9 +82,7 @@ type UseDateRangeFilterType = () => {
 export const useDateRangeFilter: UseDateRangeFilterType = () => {
     const [dateRange, setDateRange] = useState<[Dayjs, Dayjs]>([dayjs().subtract(6, 'days'), dayjs()])
 
-    const disabledDate = useCallback((currentDate) => {
-        return currentDate && currentDate < dayjs().startOf('year')
-    }, [])
+    const disabledDate = useCallback(() => false, [])
```

Comment:
I think the restriction was there for a reason.
Perhaps we should limit the last year (currentDate < now - year)

Reply:
Yeah, thats good point. It would be nice to limit aggregation range to some period. For example it might be 1 year

Reply:
Now all years are disabled off except the current and previous one